src/components/features/cart/cartItem/CartItem.tsx

```diff
@@ -1,79 +1,68 @@
 
-interface CartItemProps {
-  cartItem: CartItemType;
+interface CartItemProps extends CartItemType {
   selected: boolean;
   toggle: () => void;
-  onUpdate: () => Promise<void>;
 }
 
-function CartItem({ cartItem, selected, toggle, onUpdate }: CartItemProps) {
+function CartItem({ id, product, quantity, selected, toggle }: CartItemProps) {
+  const { remove, increase, decrease } = useCartMutations(id);
   return (
-    <S.Container data-testid={`CartItem-${cartItem.id}`}>
+    <S.Container data-testid={`CartItem-${id}`}>
       <Separator />
       <S.ActionContainer>
         <SelectBox selected={selected} onClick={toggle} />
         <S.DeleteButton
           onClick={async () => {
-            await deleteCartItem(cartItem.id);
-            onUpdate();
+            remove();
           }}
         >
           <S.DeleteButtonText>삭제</S.DeleteButtonText>
         </S.DeleteButton>
       </S.ActionContainer>
       <S.InfoContainer>
         <S.PreviewBox>
-          <S.PreviewImage
-            src={
-              isValidImageUrl(cartItem.product.imageUrl)
-                ? cartItem.product.imageUrl
-                : defaultImage
-            }
-            alt="상품 이미지"
-          />
+          {isValidImageUrl(product.imageUrl) ? (
+            <S.PreviewImage src={product.imageUrl} alt="상품 이미지" />
+          ) : (
+            <FallbackImage />
+          )}
         </S.PreviewBox>
         <S.InfoBox>
           <S.CartProductInfo>
-            <S.CartProductTitle>{cartItem.product.name}</S.CartProductTitle>
+            <S.CartProductTitle>{product.name}</S.CartProductTitle>
             <S.CartProductPrice>
-              {`${(
-                cartItem.product.price * cartItem.quantity
-              ).toLocaleString()}원`}
+              {`${(product.price * quantity).toLocaleString()}원`}
             </S.CartProductPrice>
           </S.CartProductInfo>
           <S.UpdateCartBox>
-            {cartItem.quantity === 1 ? (
+            {quantity === 1 ? (
               <IconButton
                 actionType="delete"
                 onClick={async () => {
-                  await deleteCartItem(cartItem.id);
-                  onUpdate();
+                  remove();
                 }}
               />
             ) : (
               <IconButton
                 actionType="minus"
                 onClick={async () => {
-                  await updateCartItem(cartItem.id, cartItem.quantity - 1);
-                  onUpdate();
+                  decrease(quantity);
                 }}
               />
             )}
-            <S.Text>{cartItem.quantity}</S.Text>
+            <S.Text>{quantity}</S.Text>
             <IconButton
               actionType="plus"
               onClick={async () => {
-                await updateCartItem(cartItem.id, cartItem.quantity + 1);
-                onUpdate();
+                increase(quantity);
               }}
             />
```

Comment:
앞선 리뷰를 토대로, 이 구간을 이렇게 표현할 수 있을 것 같아요!

```suggestion
              <IconButton
                actionType="delete"
                onClick={remove}
              />
            ) : (
              <IconButton
                actionType="minus"
                onClick={decrease}
              />
            )}
            <S.Text>{quantity}</S.Text>
            <IconButton
              actionType="plus"
              onClick={increase}
            />
```